Account for font-weight, font-style, and others

drwpow opened the pull request:
There’s an issue where `font-weight` isn’t carried over to the ghost element, nor is `font-style`. This results in the input not being sized properly (as the plugin assumes you are always using a non-bold, non-italic font).

While I was at it, I went ahead and added some other CSS properties that affect width that don’t seem to add any obvious overhead.

DangoDev committed:
Account for font-weight, font-style, and others

There’s an issue where `font-weight` isn’t carried over to the ghost element, nor is `font-style`. This results in the input not being sized properly (as the plugin assumes you are always using a non-bold, non-italic font).

While I was at it, I went ahead and added some other CSS properties that affect width that don’t seem to add any obvious overhead.

drwpow commented on autosize-input.js:
```diff
@@ -32,8 +32,20 @@
+                     ';font-size:' + elemStyle.fontSize +
+                     ';font-stretch:' + elemStyle.fontStretch +
+                     ';font-style:' + elemStyle.fontStyle +
+                     ';font-variant:' + elemStyle.fontVariant +
```
I know it seems silly and overkill, but one ligature or alternate number in a text field could throw off the width calculation.

antonlvovych commented on autosize-input.js:
```diff
@@ -32,8 +32,20 @@
     // Apply the `font-size` and `font-family` styles of `elem` on the
     // `ghost` element.
```
@dangodev I think you should fix comments to be consistent with your changes

yuanqing commented:
@dangodev Thanks; am OK with the change you&rsquo;re suggesting. Happy to merge it after you resolve conflicts 😃 

cc @GEEKIAM

drwpow committed:
Merge branch 'master' of github.com:dangodev/autosize-input into patch-1

# Conflicts:
#	autosize-input.js

drwpow commented:
@yuanqing Cool. Blew away my file, pulled your changes, then re-added. Also adjusted the comment above.

I also modified `autosize-input.min.js`, just uglifying it with UglifyJS2 on default settings; you’re welcome to recompress that if you preferred different settings.

yuanqing commented:
Just cut a `0.4.0`. By the way, you can rebuild `autosize-input.min.js` by doing `npm run dist`.

drwpow commented:
Oh! Duh. I glossed right over the `package.json`. Thanks!

Haraldson commented:
I’m actually struggling with this same issue right now, and I’m on 0.4.0. Help?
